import-service/src/handlers/importFileParser/index.ts

```diff
@@ -0,0 +1,61 @@
+import AWS from 'aws-sdk';
+import { S3Event } from 'aws-lambda';
+import csv from 'csv-parser';
+
+import { logger } from '../../utils/logger';
+
+export const importFileParser = async (event: S3Event): Promise<void> => {
+  const s3 = new AWS.S3({ signatureVersion: 'v4', region: 'us-east-1' });
+  logger.info('Importing file parser...');
+
+  try {
+    for (const record of event.Records) {
+      const BUCKET = record.s3.bucket.name;
+      const key = record.s3.object.key;
+
+      const params = {
+        Bucket: BUCKET,
+        Key: key,
+      };
+
+      logger.info(`CHECK PARAMS  ${params}`);
+
+      await new Promise((resolve, reject) => {
```

Comment:
Great solution! As an option consider implementation with await/async

  const s3ObjectReadableStream = s3
    .getObject({ Bucket: BUCKET_NAME, Key: objectKey })
    .createReadStream()
    .pipe(csv());

 for await (const chunk of s3ObjectReadableStream) {
    console.log(chunk);
  }